public/js/src/module/connection.js

```diff
@@ -370,31 +454,55 @@ function _encodeFormData( data ){
         .join( '&' );
 }
 
+/**
+ * @param {Document} model
+ * @param {EnketoRecord} [lastSavedRecord]
+ * @return {XMLDocument}
+ */
+function _getLastSavedInstance( model, lastSavedRecord ) {
+    if ( lastSavedRecord == null ) {
```

Comment:
We should document this fallback behavior.

Reply:
Oh, good call! Do you mean in the JSDoc, or also additional documentation?

Reply:
I meant in user-facing docs but actually I think it's non-obvious enough that it's good to have in the JSDoc briefly as well.

Reply:
Turns out those docs are _generated by_ JSDoc, but only from `/app` (the server side). Fortunately that's where the Survey type is documented, so I've added a note there, as well as here.